typescript/ecs/fargate-service-with-logging/index.ts

```diff
@@ -10,14 +10,12 @@ class WillkommenFargate extends cdk.Stack {
     const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
 
     // create a task definition with CloudWatch Logs
-    const logging = new ecs.AwsLogDriver(this, "AppLogging", {
+    const logging = new ecs.AwsLogDriver({
       streamPrefix: "myapp",
     })
 
-    const taskDef = new ecs.FargateTaskDefinition(this, "MyTaskDefinition", {
-      memoryMiB: '512',
-      cpu: '256',
-    })
+    const taskDef = new ecs.FargateTaskDefinition(this, "MyTaskDefinition")
```

Comment:
You should keep memoryLimitMiB and cpu

Reply:
@pkandasamy91 these are optional props with defaults. As a best practice we should only override them if they are different.

Reply:
In production code, I would agree with only specifying them if they differ from the defaults. for example code, I bias towards including more so that new users can easily see where a change would be made.

Reply:
they are back.